layouts/main_layout.py

```diff
@@ -7,8 +7,8 @@
 from layouts.screen_layout import create_screen_content
 import layouts.style.style as style
 from layouts.system_options_layout import create_system_options
-
-
+from layouts.calendar_layout import create_all_calendar
+from layouts.calendar_layout import change_calendar_content
```

Comment:
Maybe just `from layouts.calendar_layout import change_calendar_content, create_all_calendar`

And importing from calendar layout should be before import of files_layout (imports should be kept in alphabetical order - it's easier to read them)